app/src/main/kotlin/com/wire/android/notification/MessageNotificationManager.kt

```diff
@@ -68,14 +78,18 @@ class MessageNotificationManager @Inject constructor(private val context: Contex
     }
 
     private fun showSummaryIfNeeded(
-        oldData: List<LocalNotificationConversation>,
         newData: List<LocalNotificationConversation>,
         userId: String?
     ) {
-        if (newData.isEmpty()) notificationManager.cancel(SUMMARY_ID)
-
-        if (oldData.size <= 1 && newData.size > 1)
+        if (newData.isEmpty()) {
+            appLogger.i("$TAG removing groupSummary")
+            notificationManager.cancel(SUMMARY_ID)
+            isGroupSummaryDisplayed = false
```

Comment:
What if this notification is swiped out by user? Then it should also be changed to "false", right?